app/models/form526_submission_remediation.rb

```diff
@@ -21,6 +21,10 @@ def mark_as_unsuccessful(context)
     end
   end
 
+  def ignored_as_duplicate?
+    remediation_type == 'ignored_as_duplicate'
+  end
```

Comment:
ActiveRecord::Enum gives you this predicate method out of the box, so you can delete this method and the associated spec (`describe 'ignored_as_duplicate?'...`)

Reply:
updated!